raft/multinode.go

```diff
@@ -299,7 +300,12 @@ func (mn *multiNode) run() {
 			advancec = nil
 
 		case ms := <-mn.status:
-			ms.ch <- getStatus(groups[ms.group].raft)
```

Comment:
It still needs a temporary variable because getStatus returns a struct, not a pointer (and you can't take a pointer to the result of a function). Given that I think it's simpler as it is.